tokens/sys/shape.json

```
@@ -2,26 +2,38 @@
   "shape": {
     "zero": {
       "value": "0rem",
-      "type": "borderRadius"
+      "type": "borderRadius",
+      "description": "These elements are typically persistent, fixed to the screen and are not draggable"
+    },
+    "xhalf": {
```

I'll talk to the other engineers to confirm, but I think this should be:
```suggestion
    "xHalf": {
```